pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -365,6 +365,11 @@ func ReadConfig(config io.Reader) (Config, error) {
 		return Config{}, fmt.Errorf("no OpenStack cloud provider config file given")
 	}
 	var cfg Config
+
+	// Set default values
+	cfg.LoadBalancer.UseOctavia = true
+	cfg.LoadBalancer.InternalLB = false
```

Comment:
I understand this is improve doc so I think we might want to separate those 2 
one is to set default value (code change) and one is overall doc update.

Reply:
This is part of the intentional change, as we said in doc we will support octavia by default.

Reply:
ok, no strong opinion , but usually refactory the doc and do function changes should be separated

Reply:
@lingxiankong As its breaking change for existing users of Neutron Lbaas, IMO, separate PR would be good , where we can add release note mentioning the same, Add PR description/title accordingly, would be easy to track which PR introduced the change.
And also, if they still want to use, they just need to change the flag to false right , and we can remove the full support after 2 releases or so once we update the documentation on deprecation. Thoughts?

Reply:
ok, thanks @jichenjc @ramineni